schema/deploy/warehouse/site/data.sql

```diff
@@ -0,0 +1,45 @@
+-- Deploy seattleflu/id3c-customizations:warehouse/site/data to pg
+-- requires: seattleflu/schema:warehouse/site
+
+begin;
+
+insert into warehouse.site(identifier, details)
+    values
+        ('ChildrensHospitalBellevue',               '{"category": "clinic",     "type": "clinic"}'),
+        ('ChildrensHospitalSeattle',                '{"category": "clinic",     "type": "clinic"}'),
+        ('ChildrensSeaMar',                         '{"category": "clinic",     "type": "clinic"}'),
+        ('ColumbiaCenter',                          '{"category": "community",  "type": "workplace"}'),
+        ('Costco',                                  '{"category": "community",  "type": "workplace"}'),
+        ('DESC',                                    '{"category": "community",  "type": "shelter"}'),
+        ('FredHutchLobby',                          '{"category": "community",  "type": "workplace"}'),
+        ('Harborview',                              '{"category": "hospital",   "type": "longitudinalInpatient"}'),
+        ('HarborviewLobby',                         '{"category": "community",  "type": "workplace"}'),
+        ('HealthSciencesLobby',                     '{"category": "community",  "type": "collegeCampus"}'),
+        ('HealthSciencesRotunda',                   '{"category": "community",  "type": "collegeCampus"}'),
+        ('HUB',                                     '{"category": "community",  "type": "collegeCampus"}'),
+        ('HutchKids',                               '{"category": "community",  "type": "childcare"}'),
+        ('KaiserPermanente',                        '{"category": "clinic",     "type": "clinic"}'),
+        ('KingStreetStation',                       '{"category": "community",  "type": "publicSpace"}'),
+        ('PioneerSquare',                           '{"category": "clinic",     "type": "clinic"}'),
+        ('RetrospectiveChildrensHospitalSeattle',   '{"category": "hospital",   "type": "retrospective"}'),
+        ('RetrospectiveHarborview',                 '{"category": "hospital",   "type": "retrospective"}'),
+        ('RetrospectiveNorthwest',                  '{"category": "hospital",   "type": "retrospective"}'),
+        ('RetrospectiveUWMedicalCenter',            '{"category": "hospital",   "type": "retrospective"}'),
+        ('SeaTacDomestic',                          '{"category": "community",  "type": "publicSpace"}'),
+        ('SeaTacInternational',                     '{"category": "community",  "type": "publicSpace"}'),
+        ('SeattleCenter',                           '{"category": "community",  "type": "publicSpace"}'),
+        ('self-test',                               '{"category": "community",  "type": "self-test"}'),
+        ('StMartins',                               '{"category": "community",  "type": "shelter"}'),
+        ('swabNSend',                               '{"category": "community",  "type": "self-test"}'),
```

Comment:
I think that the metrics team is interested in separating self-test and swab-n-send, per [their community break-down table](https://docs.google.com/spreadsheets/d/1eAmWEULmgogciQnQtYEaTGFnl9hkt7snTgF0u-xNTi4/edit#gid=10263779).

Reply:
I feel like that should be looking at the site identifier itself? They are both "self-test" type

Reply:
Maybe `self-swab` is a more accurate/precise type? Both study arms swab themselves, but only the `self-test` / home test arm is actually _testing_ themselves with an at-home device (though that's on hold right now!).

But maybe introducing another term is more confusing...

Reply:
> I feel like that should be looking at the site identifier itself? They are both "self-test" type

I'm just thinking keeping self-test and swab-n-send separate will make metrics calculations much easier, seeing as the current site types are all defined by metrics-related needs.

Reply:
Taking another look at https://docs.google.com/document/d/1SqAlXuZttDXDdrRqdeSS910ecrM737MNRacBXHd6ePg/edit

They have `Swab-and-send` and `Swab-and-send with self-test` labeled as `Home` for site_type

Reply:
> seeing as the current site types are all defined by metrics-related needs

Nod, though not just metrics, but also modeling and the Nextstrain build.

Reply:
A site type of `home` makes sense to me.